packages/katello/katello/hostname-change.rb

```diff
@@ -331,12 +332,14 @@ def update_zone(zone, resolver)
         # This is fine
       end
 
-      begin
-        aaaa_record = resolver.getresource(@old_hostname, Resolv::DNS::Resource::IN::AAAA)
-        commands << "update delete #{@old_hostname} AAAA"
-        commands << "update add #{@new_hostname} #{aaaa_record.ttl} A #{aaaa_record.address}"
-      rescue Resolv::ResolvError => e
-        # This is fine
+      unless @options[:scenario] == SATELLITE_SCENARIO
+        begin
+          aaaa_record = resolver.getresource(@old_hostname, Resolv::DNS::Resource::IN::AAAA)
+          commands << "update delete #{@old_hostname} AAAA"
+          commands << "update add #{@new_hostname} #{aaaa_record.ttl} AAAA #{aaaa_record.address}"
```

Comment:
Note this was previously 'A' but I changed it to 'AAAA'. Let me know if that is not correct.

Reply:
I think it should have been AAAA all along.  Why would we delete an AAAA but add back an A... This is what was causing the bug.

Reply:
Like @jeremylenz says, it should be AAAA. I think that we shouldn't make scenario specific rules for this since it has nothing to do with a scenario. The fact that this code was triggered by someone indicates it can be useful in some scenarios.

Reply:
That makes sense to me! Updated